### DISCUSSION TITLE
Make pretalx usable for organizing Self-Organized Sessions

### DISCUSSION THREAD

**Discussion by fkusei** [pretalx/pretalx#1382] [Ideas]
I'm thinking of the following:

- Add an option to rooms to mark them as "available for Self-organized Sessions"
- Add a Track for Self-organized Sessions
- (Optional: Hide SoS-Rooms from schedule editor)
- If SoS-Track has been selected in the CfP wizard, show an additional page which allows the submitter to schedule a time slot themselves

Probably needs some more options, like "SoS need to be reviewed by an organizer" or "automatically release schedule if SoS did change". Also, we'd need to add all that scheduling stuff to the "edit my submission" workflow as well.

**Comment by rixx** [comment]
I would love to have something like this in pretalx. I think the better angle (conference agnostic, as most organisers will not know what "self-organized sessions" are!) would be: "Allow speakers to schedule their own sessions"

Building this based on track choices is interesting – I'd probably start out by making it an event-based property, to be honest. Otherwise it may require locking tracks to rooms, and we have so far avoided that successfully. (Of course, once we have implemented event-based self-scheduling, more complex versions are bound to follow.)

**Reply by wslyvh** [reply]
Allowing for self-led sessions or "Allow speakers to schedule their own sessions" would be great.

What we've currently done is to ask 'speakers' to fill in all details into the session descriptions.

It would be great if speakers were able to break down those sessions into smaller session. Keeping sessions itself consistent. We have scheduled these SLS/workshop into our regular programming though. Which makes it less relevant to link to specific tracks or rooms.

So in short:
- admin can mark a session as self-led session. Or maybe use session types for it?
- speaker (or author) can edit and break down the session to (sub-) sessions.

A reviewal process could be useful at some point, but not necessary for a first version, imo.
